EIPS/eip-2612.md

```diff
@@ -1,41 +1,42 @@
 ---
 eip: 2612
 title: permit – 712-signed approvals
```

Comment:
```suggestion
title: Permit: EIP-712-signed Approvals
```
```suggestion
title: EIP-712-signed Token Approvals
```
```suggestion
title: EIP-20 Permit Extension
```
I imagine that the title has been discussed a lot, but I'd like to give these alternative titles. The current title won't work for several reasons (it references EIP-712, so it should contain `EIP-712`, it's not in title case, and it's a really bad use of an em dash).